src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -2310,8 +2310,8 @@ TablePartitioningSupportsDistinct(List *tableNodeList, MultiExtendedOp *opNode,
 		 */
 		partitionMethod = PartitionMethod(relationId);
 
-		if (partitionMethod == DISTRIBUTE_BY_RANGE
-				|| partitionMethod == DISTRIBUTE_BY_HASH)
+		if (partitionMethod == DISTRIBUTE_BY_RANGE ||
+			partitionMethod == DISTRIBUTE_BY_HASH)
```

Comment:
I'm assuming this is because of make reindent. Could we do this in super quick separate PR?